src/apps/concave/helpers/getUserPositions.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+export const getuserPositions = async () => {
+  const gqlUrl = 'https://concave.hasura.app/v1/graphql';
```

Comment:
Hey @ibeandyy, most of your PR looks good but I'm a little nervous about this part here since I don't think the GQL server will stand up to our traffic. I know what you're trying to do though, and I think we can get the `Lock` events directly from our Ethers instance. I'll merge your PR and try to make that change (its not documented, so I don't want you to have to struggle with figuring out on your own 🙏 )